index.html

```diff
@@ -112,36 +113,41 @@ <h3>Meeting Minutes</h3>
     <li>Anthony Mace (general member 2019-2020)</li>
-    <li><a href="https://twitter.com/AndrewPett1" target="_blank">Andrew Pett</a> (secretary 2019-2020)</li>
-    <li><a href="https://twitter.com/steveh7" target="_blank">Steve Hoeksema</a> (treasurer 2016-2020)</li>
-    <li><a href="https://twitter.com/ur5us" target="_blank">Juri Hahn</a> (general member 2016-2017, 2018-2019)</li>
-    <li><a href="https://twitter.com/nz_cub3y" target="_blank">Charles Peach</a> (general member 2016-2018, president 2018-2019)</li>
-    <li><a href="https://twitter.com/containsulfite" target="_blank">Mai Nguyen</a> (secretary 2018-2019)</li>
-    <li><a href="https://twitter.com/charlieablettnz" target="_blank">Charlie Ablett</a> (general member 2018-2019)</li>
-    <li><a href="https://twitter.com/ioquatix" target="_blank">Samuel Williams</a> (general member 2018-2019)</li>
-    <li><a href="https://twitter.com/a_wagener" target="_blank">Amanda Wagener</a> (general member 2017-2018 and 2014-2015, president 2016-2017, secretary 2015-2016)</li>
-    <li><a href="https://twitter.com/danielfone" target="_blank">Daniel Fone</a> (secretary 2016-2018)</li>
-    <li><a href="https://twitter.com/psdavey" target="_blank">Patrick Davey</a> (general member 2017-2018)</li>
-    <li><a href="https://twitter.com/_maxhungry" target="_blank">Max Hung</a> (general member 2017-2018)</li>
-    <li><a href="https://twitter.com/raquelxmoss" target="_blank">Raquel Moss</a> (president 2017-2018, general member 2016-2017)</li>
-    <li><a href="https://twitter.com/merxplat" target="_blank">Merrin Macleod</a> (secretary 2014-2015, general member 2016-2017, president 2020-2021)</li>
-    <li><a href="https://twitter.com/eoinkelly" target="_blank">Eoin Kelly</a> (treasurer 2014-2016)</li>
-    <li><a href="https://twitter.com/parndt" target="_blank">Philip Arndt</a> (president 2014-2016, treasurer 2020-2021)</li>
-    <li><a href="https://twitter.com/megahbite" target="_blank">Megan Bowra-Dean</a> (general member 2015-2016)</li>
+    <li>Andrew Pett (secretary 2019-2020)</li>
+    <li>Steve Hoeksema (treasurer 2016-2020)</li>
+    <li>Juri Hahn (general member 2016-2017, 2018-2019)</li>
+    <li>Charles Peach (general member 2016-2018, president 2018-2019)</li>
+    <li>Mai Nguyen (secretary 2018-2019)</li>
+    <li>Charlie Ablett (general member 2018-2019)</li>
+    <li>Samuel Williams (general member 2018-2019)</li>
+    <li>Amanda Wagener (general member 2017-2018 and 2014-2015, president 2016-2017, secretary 2015-2016)</li>
+    <li>Daniel Fone (secretary 2016-2018)</li>
+    <li>Patrick Davey (general member 2017-2018)</li>
+    <li>Max Hung (general member 2017-2018)</li>
+    <li>Raquel Moss (president 2017-2018, general member 2016-2017)</li>
+    <li>Merrin Macleod (secretary 2014-2015, general member 2016-2017, president 2020-2021)</li>
+    <li>Eoin Kelly (treasurer 2014-2016)</li>
+    <li>Philip Arndt (president 2014-2016, treasurer 2020-2021, treasurer 2022-2023)</li>
```

Comment:
The final phrase should be `treasurer 2022-2024`

Reply:
@sarahhay7 Yes, right. Also should I merge them, say `Philip Arndt (president 2014-2016, treasurer 2020-2021, 2022-2024)`?

Reply:
@sarahhay7 I guess there will be more previous members coming back doing the same role in the future as the year goes on.

Reply:
@sarahhay7 which will be nice.

Reply:
Yep, `Philip Arndt (president 2014-2016, treasurer 2020-2021, treasurer 2022-2024)` would be what I would put